core/tools/generate_names.cc

```diff
@@ -182,6 +182,18 @@ NameDef names[] = {
     {"prefix"},
     {"to"},
 
+    {"mattrAccessor", "mattr_accessor"},
+    {"mattrReader", "mattr_reader"},
+    {"mattrWriter", "mattr_writer"},
+    {"cattrAccessor", "cattr_accessor"},
+    {"cattrReader", "cattr_reader"},
+    {"cattrWriter", "cattr_writer"},
+    {"instanceReader", "instance_reader"},
+    {"instanceWriter", "instance_writer"},
+    {"instanceAccessor", "instance_accessor"},
+    {"instancePredicate", "instance_predicate"},
+    {"classAtribute", "class_attribute"},
```

Comment:
Not that it matters a lot, but there is a typo here: `classAtribute` -> `classAttribute`